File: runtimes/cron.go
package main

import (
	"fmt"
	"os"
	"strings"
)

func main() {
	envs := make(map[string]string)
	for _, entry := range os.Environ() {
		if strings.HasPrefix(entry, "ESERVELESS_") {
			parts := strings.Split(entry, "=")
			envs[parts[0]] = parts[1]
		}
	}
	Ticker(envs)
	fmt.Println(envs)
}

File: runtimes/cron.js
const functions = require('./functions') 

const envs = Object
    .keys(process.env)
    .filter(name => name.startsWith('ESERVELESS_'))
    .reduce((envs, name) => {
            envs[name] = process.env[name];
            return envs
        }, {});
    
functions.Ticker(envs)
